scripts/release.sh

```
@@ -0,0 +1,70 @@
+# Release script for instructor-stream-js workspace
+# Handles version bumping, changelog, git operations, and NPM publishing
+
+set -e
+
+echo "🚀 Starting release process..."
+echo "================================"
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
```

Shellcheck (0.10.0)

[warning] YELLOW appears unused. Verify use (or export if used externally).

(SC2034)